release.nix

```diff
@@ -1,103 +1,21 @@
-    th-expand-syns = self.callHackageDirect {
-      pkg = "th-expand-syns";
-      ver = "0.4.6.0";
-      sha256 = "1l2g98jfg86blp8mkkvzh90h557l5qklw1nn045zqb5am8977dgq";
-    } {};
-    ChasingBottoms = self.callHackageDirect {
-      pkg = "ChasingBottoms";
-      ver = "1.3.1.8";
-      sha256 = "0klxmb6pgl2xv5206gn2m3n1di2aidkfyi5rlqcfdx5qvpbnhl19";
-    } {};
-    haskell-src-meta = self.callHackageDirect {
-      pkg = "haskell-src-meta";
-      ver = "0.8.5";
-      sha256 = "1dhncvsyv2kc8x18gvr7if4pr7vvypl0lr450jaaj3xj7rly3lwv";
-    } {};
-    haskell-src-exts = self.callHackageDirect {
-      pkg = "haskell-src-exts";
-      ver = "1.22.0";
-      sha256 = "1w1fzpid798b5h090pwpz7n4yyxw4hq3l4r493ygyr879dvjlr8d";
-    } {};
-    constraints = self.callHackageDirect {
-      pkg = "constraints";
-      ver = "0.11";
-      sha256 = "0xi2p57hsdy31f8a4isxxp1zgv8m7a26c586jlz8p2rmk0ypw3pj";
-    } {};
-    constraints-extras = doJailbreak super.constraints-extras;
-    some = doJailbreak super.some;
-  };
-  ghcs = rec {
-    ghc802 = nixos1809.haskell.packages.ghc802;
-    ghc844 = nixos1809.haskell.packages.ghc844;
-    ghc865 = nixos2003.haskell.packages.ghc865;
-    ghc884 = nixos2003.haskell.packages.ghc884.override {
-      overrides = dep-sum-overrides;
-    };
-    ghc8101 = nixos2003.haskell.packages.ghc8101.override {
-      overrides = ghc810-overrides;
-    };
-    ghc8102 = unstable.haskell.packages.ghc8102;
-    ghc8101_aeson15 = nixos2003.haskell.packages.ghc8101.override {
-      overrides = self: super: ghc810-overrides self super //
-      { assoc = doJailbreak super.assoc;
-        strict = self.callHackageDirect {
-          pkg = "strict";
-          ver = "0.4";
-          sha256 = "0sl9mfpnyras2jlpjfnji4406fzp0yg2kxfcr22s3zwpir622a97";
-        } {};
-        these = self.callHackageDirect {
-          pkg = "these";
-          ver = "1.1.1.1";
-          sha256 = "1i1nfh41vflvqxi8w8n2s35ymx2z9119dg5zmd2r23ya7vwvaka1";
-        } {};
-        aeson = doJailbreak (self.callHackageDirect {
-          pkg = "aeson";
-          ver = "1.5.2.0";
-          sha256 = "0rz7j7bcj5li2c5dmiv3pnmbs581vzkl9rbx9wq2v06f4knaklkf";
-        } {});
-        hashable-time = doJailbreak super.hashable-time;
-        Diff = self.callHackageDirect {
-          pkg = "Diff";
-          ver = "0.4.0";
-          sha256 = "1phz4cz7i53jx3d1bj0xnx8vpkk482g4ph044zv5c6ssirnzq3ng";
-        } {};
-        doctest = dontCheck (self.callHackageDirect {
-          pkg = "doctest";
-          ver = "0.16.3";
-          sha256 = "0rm91akq3d4b8xa127dklgd1vg2x9xv962pg98i7xhgnllp6i5r3";
-        } {});
-        quickcheck-instances = self.callHackageDirect {
-          pkg = "quickcheck-instances";
-          ver = "0.3.23";
-          sha256 = "1baqh2harkcx7igqmk6p040vmchy30wnh1crwwvzcxqv22iwyfrw";
+let pkgs = import ./nixpkgs/23.11 {};
+    hpkgs = {
+      overrides = self: super: {
+        dependent-sum-template = self.callHackageDirect {
```

Comment:
@ali-abrar  Might be able to just do
https://github.com/NixOS/nixpkgs/blob/e22b1a7d7b0e338cca6932ab5a3f16e20fcf1c32/pkgs/development/haskell-modules/configuration-ghc-9.8.x.nix#L91

Reply:
yeah, probably. but i already merged ;)

Reply:
RIght, I meant it more as a FYI for any other repos with a big blob of release.nix that can now be aggressively pruned down

Reply:
thanks for all the PRs btw